scripts/src/tests/services/orders.spec.ts

```diff
@@ -710,4 +711,18 @@ describe("test v2 orders", async () => {
 		expect(history1.length).toBe(history2.length);
 		history1.forEach(o => expect(history2).toContain(o));
 	});
+
+	test("create a cross-app order shaharsol", async () => {
+		const senderApp = await helpers.createApp("sender-app");
+		const receiverApp = await helpers.createApp("receiver-app");
+		const deviceId1 = `device_${ generateId() }`;
+		const senderWalletAddress = `wallet-${ generateId() }`;
+		const receiverWalletAddress = `wallet-${ generateId() }`;
```

Comment:
Use the `generateRandomString()` utility function here too

Reply:
Keypair.random or copy from somewhere else in the unittests

Reply:
done